app/viewmodels/journeys/taskList/TradeJourneyStatusesViewModel.scala

```diff
@@ -48,14 +48,26 @@ object TradeJourneyStatusesViewModel {
     val isAbroadAnswered = tradesJourneyStatuses.getStatusOrNotStarted(Abroad).isCompleted
     val incomeRow        = buildRow(Income, dependentJourneyIsFinishedForClickableLink = isAbroadAnswered)
 
+    val expensesRows: Seq[SummaryListRow] = buildExpensesCategories
+    val expensesAllCompleted: Boolean     = expensesRows.forall(checkIfRowIsCompleted)
+
+    val capitalAllowanceRows: Seq[SummaryListRow] = buildCapitalAllowances(tradesJourneyStatuses, taxYear)
+    val capitalAllowanceAllCompleted: Boolean     = capitalAllowanceRows.forall(checkIfRowIsCompleted)
+
+    val adjustmentsRow = buildRow(Adjustments, dependentJourneyIsFinishedForClickableLink = capitalAllowanceAllCompleted && expensesAllCompleted)
```

Comment:
Needs to check income as well because income could be incomplete meaning expenses would be empty but still pass true

Reply:
Done :)